lib/perl/Genome/Model/Tools/Vcf/FinalPassOnly.pm

```diff
@@ -0,0 +1,72 @@
+package Genome::Model::Tools::Vcf::FinalPassOnly;
+
+use Genome;
+use strict;
+use warnings;
+use Genome::File::Vcf::Reader;
+use Genome::File::Vcf::Writer;
+
+class Genome::Model::Tools::Vcf::FinalPassOnly {
+    is => 'Command',
```

Comment:
New commands should probably use `Command::V2`.  There's less of an effect here since none of the parameters are objects, though.